docs/nmap.dtd

```diff
@@ -151,6 +151,8 @@
 >
 
 <!-- these elements are written by output.c:write_xml_initial_hostinfo() -->
+<!ELEMENT hosthint (status,address,hostnames)* >
```

Comment:
We write this in `scan_engine.c:ultrascan_host_pspec_update()` might be worth moving the comment above this below and then having a separate comment to explain where hosthint is written